ChainingDisjointTests.dfy(49,2): Error: a postcondition could not be proved on this return path
ChainingDisjointTests.dfy(48,14): Related location: this is the postcondition that could not be proved
ChainingDisjointTests.dfy(42,22): Related location: this proposition could not be proved
ChainingDisjointTests.dfy(58,13): Error: assertion might not hold
ChainingDisjointTests.dfy(58,18): Error: assertion might not hold
ChainingDisjointTests.dfy(58,23): Error: assertion might not hold

Dafny program verifier finished with 4 verified, 4 errors
